pkg/controller/infrastructure/actuator_delete.go

```diff
@@ -59,12 +59,18 @@ func (a *actuator) Delete(ctx context.Context, log logr.Logger, infra *extension
 	azureClientFactory := NewAzureClientFactory(a.Client())
 	resourceGroupExists, err := infrastructure.IsShootResourceGroupAvailable(ctx, azureClientFactory, infra, config)
 	if err != nil {
-		return err
+		if azureclient.IsAzureAPIUnauthorized(err) {
+			log.Info("failed to check resource group availability due to invalid credentials", err)
```

Comment:
According to https://github.com/gardener/gardener/blob/master/docs/development/logging.md, log messages should be capitalized.